src/test/java/uk/gov/hmcts/cmc/claimstore/services/interest/InterestCalculationServiceInterestUpToNowTest.java

```diff
@@ -53,7 +53,7 @@ public void interestUpToNowFrom12345PoundsOf75PercentFor500DaysShouldEqualExpect
         BigDecimal calculated = service.calculateInterestUpToNow(
             MORE_POUNDS, SEVENTY_FIVE_PERCENT_INTEREST, FIVE_HUNDRED_DAYS_AGO
         );
-        assertThat(calculated).isEqualByComparingTo("12674.81");
+        assertThat(calculated).isEqualByComparingTo("12683.22");
```

Comment:
It is Christmas time, you adding more money to people :)

Reply:
ho ho ho ;)